cms/static/require-config.js

```diff
@@ -27,6 +27,7 @@ require.config({
         "jquery.immediateDescendents": "coffee/src/jquery.immediateDescendents",
         "datepair": "js/vendor/timepicker/datepair",
         "date": "js/vendor/date",
+        "moment": "js/vendor/moment.min",
```

Comment:
@sarina ok, i experimented again and I think I came up with a different solution. Here's the breakdown:

```javascript
date = new Date(); // Mon Jun 01 2015 08:20:10 GMT+0200 (CEST)
utc = new Date(date.getUTCFullYear(), date.getUTCMonth(), date.getUTCDate(), date.getUTCHours(), date.getUTCMinutes(), date.getUTCSeconds()); // Mon Jun 01 2015 06:20:10 GMT+0200 (CEST)

// This gets stored in the cookie
utcUnix = utc.getTime(); // 1433132410000

// This is to format and display the unix timestamp on the page
new Date(utcUnix).toString('MM/dd/yyyy HH:mm'); // 06/01/2015 06:20"
```
Note that `utc` is not really a UTC date (it still has the `GMT+0200` timezone) but at least *date.js* can parse it.

I need to point out though that `date.js` is an [incredibly old](https://github.com/datejs/Datejs/blob/master/build/date.js) library

```javascript
/**
 * @version: 1.0 Alpha-1
 * @author: Coolite Inc. http://www.coolite.com/
 * @date: 2008-05-13
```
which is currently not supported (although the [github page](https://github.com/datejs/Datejs) claims that steps to revive it are taking place), as you can also tell by its [website](http://datejs.com/).

On the other hand [moment.js](http://momentjs.com/) is very much alive, supported and updated. I wasn't very keen on adding an additional dependency, but maybe it's worth considering keeping it as a way to have a reliable datetime library ready to use?

Let me know what you think, thanks!